mvpa2/misc/plot/flat_surf.py

```diff
@@ -92,17 +112,34 @@ def flat_surface2xy(surface):
     '''
 
     s = surf.from_any(surface)
+    face_normals=s.face_normals
```

Comment:
ideally for new code we should follow pep8.  spaces around '=', and preceding ',' here and below .  Since you like IDEs -- give a shot to PyCharm -- it would do highlighting